glitter/pages/admin.py

```diff
@@ -57,6 +57,10 @@ def get_fields(self, request, obj=None):
         if not getattr(settings, 'GLITTER_SHOW_LOGIN_REQUIRED', False):
             fields.remove('login_required')
 
+        # Show glitter tags if it's set to show.
+        if getattr(settings, 'GLITTER_PAGES_TAGS', False):
```

Comment:
Looking at the lines above, shall we keep the adding/removing consistent?

I'd possibly suggest removing, so we can position fields as we want in `fields`, and just remove the bits we don't want.